src/others/components/MultiTab.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+
+interface MultiTabProps {
+  labels: string[];
+}
+
+// TODO: This one still needs styling
```

Comment:
Actually, this one became so complicated when customised, that I'd go with
a) building a custom, simple one
b) take the simplest MUI tabs